packages/errors/src/errors.ts

```diff
@@ -279,7 +279,23 @@ export const AllCypressErrors = {
 
         https://on.cypress.io/parallel-disallowed`
   },
-  CLOUD_PARALLEL_GROUP_PARAMS_MISMATCH: (arg1: {runUrl: string, parameters: any}) => {
+  CLOUD_PARALLEL_GROUP_PARAMS_MISMATCH: (arg1: {runUrl: string, parameters: any, payload: any }) => {
+    let params: any = arg1.parameters
+
+    if (arg1.payload?.differentParams) {
+      params = {}
+
+      _.map(arg1.parameters, (value, key) => {
+        if (key === 'specs') {
```

Comment:
```suggestion
        if (key === 'specs'  && arg1.payload.differentSpecs) {
```
do we need to include `differentSpecs` at all if there is nothing to show the user?

Reply:
do we want to show different error messages depending on the payload? If there are no `differentSpecs`  we show `specs` with all the specs ran but if there're different specs, we are showing it

Reply:
If the payload is meant to show the differences on why this errored, I think providing: `differentSpecs: []` is a little confusing / doesn't provide much but can go either way.

Reply:
Going to remove that field in case it is empty! you're right! it can be confusing